packages/components/src/components/post-popovercontainer/post-popovercontainer.tsx

```diff
@@ -59,6 +59,11 @@ export class PostPopovercontainer {
    */
   @Prop() readonly placement?: Placement = 'top';
 
+  /**
+   * Gap between the edge of the page and the popover
+   */
+  @Prop() readonly edgeGap?: number = 8;
```

Comment:
Should we really set the default value to something else than `0`?
It will possibly change the default behaviour for some projects.

Reply:
This is the value that was used before, so anything else would mean that it changes for projects that don't specify. It was hardcoded before.